### PR TITLE
Refs task 4087 Create project

### PR DESCRIPTION
**Pull request by duccnv-1684**
https://edu-redmine.framgia.vn/issues/4087

### CODE REVIEW

**Comment by duccnv-1684** [comment]
ready

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,28 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 28
+    defaultConfig {
+        applicationId "com.example.ducnguyen2102.f_talk"
```
Comment (DoanVanToan) [inline app/build.gradle]: rename app id and package name to com.framgia.ftalk

File: app/src/main/java/com/example/ducnguyen2102/f_talk/HomeActivity.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.example.ducnguyen2102.f_talk;
```
Comment (DoanVanToan) [inline app/src/main/java/com/example/ducnguyen2102/f_talk/HomeActivity.java]: rename package to com.framgia.ftalk

File: app/src/main/res/layout/activity_home.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".HomeActivity">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Hello World!"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/activity_home.xml]: remove this line,
hardcode

**Comment by DoanVanToan** [comment]
commented

**Comment by duccnv-1684** [comment]
ready

**Review by DoanVanToan — APPROVED** [review]

**Pull request merged**